user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube/Opening_DataMiner_Cube.md

```diff
@@ -38,6 +38,30 @@ DataminerCube.exe
 
 Below you can find more information on the different options available to manage the start window of the DataMiner Cube desktop app, available from DataMiner 10.0.9 onwards.
 
+### Selecting your Cube update track
+
+From DataMiner 10.2.0 \[CU3]/10.2.6 onwards, Cube can automatically update to a more recent version than the DataMiner version installed on the server. This way you can use the latest Cube features as soon as they are released without having to wait for a server upgrade.
+
+In the Cube start window, you can select which Cube update track should be used:
+
+1. Click the cogwheel icon in the lower right corner of the start window.
+
+1. Select *Settings*.
+
+1. In the *Settings* dialog, select the update track you want to use:
+
+   - **Development**: Use the latest available development version.
+   - **Preview**: Use a preview of the latest DataMiner Cube version, even if it has not been released yet.
```

Comment:
Note: the _development_ and _preview_ tracks are only available to Skyline employees.